ADDRESSING.md

```diff
@@ -22,20 +22,17 @@
 
 If no native protocol handler is available, redirect to path-based IPFS address at a gateway. The implementation should detect if a local IPFS node is available. In web browser contexts where a local IPFS node is present, use [subdomain gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `localhost`. If not, use public one such as `dweb.link`. 
 
-In either case, point at IPFS path first to ensure gateway takes care of CID normalization into a DNS-safe form:
+In either case, point at IPFS path first to ensure gateway takes care of CID normalization into a DNS-safe form ([docs](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway)):
 
 ```
 Native URI            – HTTP Gateway                       (– Internal normalization done by HTTP Gateway)   
 
 ipfs://{cid}          → https://dweb.link/ipfs/{cid}        → HTTP301 → https://{dns-safe-cid}.ipfs.dweb.link
-ipns://{libp2p-key}   → https://dweb.link/ipns/{libp2p-key} → HTTP301 → https://{dns-safe-cid}.ipns.dweb.link
-```
 
-When DNSLink hostname is used, redirect to subdomain only with `localhost`, use original HTTP URL otherwise:
```

Comment:
why remove? would be better if each entry had descriptive comment i think? (for future work, not in this pr)